Update and migrate translating page from wiki (#16577)
nvaccess/nvda: projectDocs/translating/readme.md

## Translation Mailing List
Translators should subscribe to the [NVDA translations mailing list](https://groups.io/g/nvda-translations) hosted at Groups.IO.

It is an English low traffic list devoted for the discussion of translation. 
Important messages that relate to translators will also be sent here, e.g. before official NVDA releases, to remind translators to make sure their localization is up to date.

## Important Dates

Translators should ensure their translation is up to date during the beta period before each new release, in order for it to be included in the upcoming final release.
A reminder will be posted on the translation mailing list giving two weeks notice of the translation string freeze.
Work submitted after the translation string freeze date will not be included in the upcoming release.
For further information please see the [Release Process page](https://github.com/nvaccess/nvda/blob/master/projectDocs/community/releaseProcess.md).

## Translation Status

You can view [Crowdin](https://crowdin.com/project/nvda) for an up to date report on the status of translating the NVDA interface.

> If you would like to improve or would like to work on a new language, please write to the [NVDA translations mailing list](https://groups.io/g/nvda-translations).

Author: @michaelDCurran is this more accurate?

```suggestion
If you would like to improve an existing language, please write to NV Access via info@nvaccess.org.
```

> If you would like to improve or would like to work on a new language, please write to the [NVDA translations mailing list](https://groups.io/g/nvda-translations).

Reviewer: I think I would prefer to keep this on the translations list. There are more people that can help.  Perhaps we can give a couple of trusted community members full language coordinator permissions on Crowdin for all languages, thus they will be able to assign permissions for others. As for the translation system itself: we will not be taking any new translations for documentation on that system. It will be moving to po files on Crowdin  sometime this year. If I miss anything on the translation list for now,  trust that a community member will nudge me :)

> If you would like to improve or would like to work on a new language, please write to the [NVDA translations mailing list](https://groups.io/g/nvda-translations).

Reviewer 2: Recently, I have had to remind NV Access of pending requests of people wanting to translate new language, hence my proposal to contact you directly.

If you prefer delegating the possibility to create new languages and grant approve rights to trusted members, that's another path. And I am completely fine with that.

The translation status of user documentation (User Guide and Changes) can only be checked by translators.

## New Localization
Start by subscribing to the translation list above, so that you can get help and advice.

The current process for translation is split between multiple processes:
- Crowdin for the NVDA interface
- The legacy SVN translation system for the User Guide and Changes files

> - GitHub for Character Descriptions, Symbols and Gestures

Author: @michaelDCurran communicated that these files should be updated via GitHub PRs in https://github.com/nvaccess/nvda/discussions/15014

@michaelDCurran - can you confirm that we no longer support updating these files in SVN?

As mentioned further down, the guides for translating files need to be updated. 
This PR is just refreshing the overall wiki page. 
The wiki pages for translating these files will be updated and migrated after this PR.

> - GitHub for Character Descriptions, Symbols and Gestures

Reviewer: I never specifically removed support for those files from the system, I was more just uncertain  how that worked / if that code still functioned. I think technically they could still be submitted that way. But as soon as we move userGuide / changes to po files, then I would not want to keep the system around only for symbols / gestures. Thus  as recommended they should be now submitted in PRs.

Read [Files to be Localized](#files-to-be-localized) to learn the translation for process for these.

## Improving an Existing Localization
You should contact the existing maintainer of your language, and discuss your suggestions and changes.
Together you should be able to agree on the best translation and terms to be used, and the necessary changes can be made.

You can send an email to the translation mailing list to find out the correct person. 

> If your language is no longer maintained, you can request to be the new maintainer for the language.

Author: ```suggestion
If your language is no longer maintained, you can request to be the new maintainer for the language by contacting NV Access via info@nvaccess.org.
```

> If your language is no longer maintained, you can request to be the new maintainer for the language.

Reviewer: Again, I'd prefer to keep this on the translations list.

> If your language is no longer maintained, you can request to be the new maintainer for the language.

Author: ```suggestion
If your language is no longer maintained, you can request to be the new maintainer for the language via the [NVDA translations mailing list](https://groups.io/g/nvda-translations).
```
